learning_resources/migrations/0045_learningresource_next_start_date.py

```diff
@@ -14,7 +14,7 @@ def assign_next_start_date(apps, schema_editor):
     for resource in LearningResource.objects.filter(
         resource_type__in=["course", "program"]
     ).iterator():
-        load_next_start_date(resource)
+        load_next_start_date_and_prices(resource)
```

Comment:
This will break if this migration is run and there is data because the prices column is not added until 0057. I think it's better to just delete the `assign_next_start_date` part of the migration

Reply:
Good catch, I didn't even notice that this migration had been affected.

Reply:
I removed that function from the migration